Dependencies/xr/Include/XR.h

```diff
@@ -46,6 +53,7 @@ namespace xr
         } Orientation;
     };
 
+    const float FLOAT_COMPARISON_THRESHOLD = .02f;
```

Comment:
A few points:
- I don't think this is part of the contract and should not be exposed through XR.h.
- Our existing style leads floats with numerals (i.e. `0.02f`) for consistency.
- I believe this could be `constexpr`.
- nit: In my experience, the value used for this purpose is usually called epsilon, signifying, "a number small enough that we don't care about it."

Reply:
Will move to the individual .cpp files.  I had it here so it could be shared between xr.mm and xr.cpp